source-mongodb/discovery.go

```diff
@@ -24,10 +24,12 @@ const idProperty = "_id"
 const (
 	metaProperty    = "_meta"
 	deletedProperty = "deleted"
+	opProperty      = "op"
 )
 
 type documentMetadata struct {
-	Deleted bool `json:"deleted,omitempty" jsonschema:"title=Delete Flag,description=True if the document has been deleted, unset otherwise."`
+	Deleted bool `json:"deleted,omitempty" jsonschema:"title=Delete Flag (deprecated, use op instead),description=True if the document has been deleted, unset otherwise."`
```

Comment:
What would it take to completely get rid of this `deleted` property? I think the longer we wait to remove it, the harder it will be since more captures will have been created with it. But also going forward it's going to be confusing to have both `op: d` and `delete: true` in the documents - I would wonder what the difference is, which I could trust, etc. I'm assuming we would need to evaluate current production uses of the connector to see if a version bump and/or migration would be needed.